sdk/storage/Azure.Storage.Files.DataLake/tests/DataLakeSasBuilderTests.cs

```diff
@@ -212,15 +212,13 @@ public async Task DataLakeSasBuilder_DirectoryRawPermissions_Exists()
                 startsOn: null,
                 expiresOn: Recording.UtcNow.AddHours(1));
 
-            // Set the Resource = "d" and the DirectoryDepth = 1
             DataLakeSasBuilder dataLakeSasBuilder = new DataLakeSasBuilder
             {
                 StartsOn = Recording.UtcNow.AddHours(-1),
                 ExpiresOn = Recording.UtcNow.AddHours(1),
                 FileSystemName = test.FileSystem.Name,
                 Path = directory.Path,
-                IsDirectory = true,
-                DirectoryDepth = 1
+                IsDirectory = true
```

Comment:
Did you want to add a test to test that ensure state method with a number of different object paths?

Reply:
Yes I did! I guess I erased it before I pushed the PR. Thanks for the catch